data/remnant missions.txt

```diff
@@ -1252,3 +1252,214 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Jump Drive (Broken)"
+	minor
+	invisible
+	landing
+	to offer
+		has "remnant blood test pure"
+	to complete
+		has "Jump Drive (Broken)"
+	source
+		attributes "remnant"
+	destination 
+		attributes "remnant"
+	on offer
+		conversation
+			`The Remnant seem very interested in acquiring alien technology. Despite its broken nature, they might be interested in buying this partially functional Jump Drive. Do you want to ask someone about it?`
+			choice
+				`	(No.)`
+					defer
+				`	(Yes.)`
+			`	You stop by the outfitters and ask one of the engineers if they would be interested. They immediately perk up and offer you <payment> in exchange for the broken jump drive.`
+			choice
+				`	"Sure, that sounds good."`
+					goto end
+				`	"Can you offer me more than that?"`
+					goto more
+				`	"How will my ship leave here without my jump drive?"`
+			
+			`	The engineer looks at you incredulously "You were relying on this for your interstellar travel? You are lucky you made it this far, indeed." He shakes his head in amusement, then continues "If you need a replacement, we have standard hyperdrives available in the outfitter."`
+			choice
+				`	"Okay, I accept your offer."`
+					goto end
+				`	"Sorry, I think I'll hold on to this for a while longer."`
+					goto refuse
+			
+			label refuse
+			`	He considers this, and sings, "If you wish. Our offer stands, whenever you choose to return."`
+				defer
+			
+			label more
+			`	"We do not have an infinite supply of wealth" he sings. "And we must budget our rewards in the expectation that we will need scores or even hundreds of these to unlock the secrets of Jump Drives." He pauses a moment, then continues. "While we cannot offer you more funds, we can offer you early access to the results: Contributing to our research could earn you a place among our test pilots someday when our researcher's work is complete."`
+			choice
+				`	"Okay, that sounds worthwhile then."`
+					goto end
+				`	"Sorry, I'm not interested right now."`
+					goto refuse
+			
+			label end
+			`	The Remnant engineers carefully remove the broken jump drive and whisk it away down into the tunnels. "Good hunting!" He sings as he hands you your payment. "We encourage you to salvage more of these whenever you have the opportunity."`
+				accept
+	
+	on accept
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+		fail "Remnant: Broken Jump Drive 1"
```

Comment:
Since you are failing the mission that this `MissionAction` belongs to, you can simply write `fail`.